bots/running-a-hubot-bot/README.md

```diff
@@ -58,7 +58,7 @@ export ROCKETCHAT_USE_SSL=true
 
 Adjust the content to fit your server and user credentials. Make sure `mybotuser`
 has **BOT role** on the server, if you don't know what that means,
```

Comment:
Something like:

Assign the new user a Role of 'Bot'. This role has a special set of permissions for Bots.

You can give your bot user extra Permissions as required.

See admin/permissions/

Reply:
This part of text wasn't modified in the PR. I will apply your mention in the upcoming PR.